guides/Circle CI 2.md

```diff
@@ -0,0 +1,195 @@
+# Circle CI 2.0 configurations for Rails project
```

Comment:
# Circle CI 2.0 Configuration for Rails

The following is a recommended setup for Circle CI 2.0 with any rails project. This guide will walk you through the different parts of the setup to get you familiar with the process.

If you just want the script, it's [here](#the-end-result)